WordPress/src/jetpack/java/org/wordpress/android/ui/accounts/login/components/LoopingText.kt

```diff
@@ -20,11 +21,13 @@ import org.wordpress.android.ui.accounts.login.LocalPosition
 import org.wordpress.android.ui.compose.theme.AppTheme
 import org.wordpress.android.util.extensions.isOdd
 
-private val fontSize = 40.sp
-private val lineHeight = fontSize / 100 * 105 // last value = % of fontSize
+private const val FIXED_FONT_SIZE = 40
 
 @Composable
 private fun LargeTexts() {
+    val fontSize = (FIXED_FONT_SIZE / LocalDensity.current.fontScale).sp
+    val lineHeight = fontSize / 100 * 105 // last value = % of fontSize
```

Comment:
Very minor nitpick:

The order of operations (and comment) are slightly confusing (I know the precedence, but at first glance it almost seems like 105 could be in the denominator).

I was thinking maybe we could change this to:

```suggestion
    val lineHeight = fontSize * 1.05 // set line height to 5% larger than the font size
```

Wdyt?

Reply:
> The order of operations (and comment) are slightly confusing (I know the precedence, but at first glance it almost seems like 105 could be in the denominator).

Totally agree @mkevins, I've been thinking of updating this line of code each time I looked at it 😅 .

Thank you for suggesting the improvement 🙇